content/docs/examples/advanced-gateways/egress-gateway-mtls-origination/index.md

```diff
@@ -298,10 +298,10 @@ to hold the configuration of the NGINX server:
     {{< /text >}}
 
 1.  Use the deployed [sleep]({{< github_tree >}}/samples/sleep) pod to send requests to the NGINX server.
-    Since the `nginx.example.com` host does not exist, the DNS cannot resolve the hostname. The following command uses the
+    Since `nginx.example.com` does not actually exist and therefore DNS cannot resolve it, the following command uses the
     `--resolve` option of `curl` to resolve the hostname manually. You can provide any IP to the `--resolve` option,
-    except for `127.0.0.1`. If you use, for example, `1.1.1.1`, Istio routes the request correctly to your NGINX server.
-    Normally, a DNS entry exists for the destination hostname and you must not use the `--resolve` option of `curl`.
+    except for `127.0.0.1`. If you use `1.1.1.1`, for example, Istio will route the request correctly to your NGINX server.
```

Comment:
"If ..., Istio routes" is _zero-conditional_, see https://www.grammarly.com/blog/conditional-sentences/:

> Zero conditional sentences express general truths—situations in which one thing always causes another. When you use a zero conditional, you’re talking about a general truth rather than a specific instance of something. Consider the following examples:

> If you don’t brush your teeth, you get cavities.
When people smoke cigarettes, their health suffers.

> There are a couple of things to take note of in the above sentences in which the zero conditional is used. First, when using the zero conditional, the correct tense to use in both clauses is the simple present tense. A common mistake is to use the simple future tense.

> Incorrect:
> When people smoke cigarettes, their health will suffer .

Reply:
Yes, I guess that's a little awkward. I'll change the sentence to:

For example, `1.1.1.1` will work.

Reply:
@frankbu the problem is with the future tense/first-conditional. As far as I understand "will" should not be used in documentation.

Reply:
I think we may be over analyzing ... in this context "will work" is just an informal way of saying "is an acceptable value". Anyway, maybe restructuring the previous sentence would make it flow better:

The IP value passed to the `--resolve` option (`1.1.1.1` below) is not significant. Any value other than `127.0.0.1` can be used.

Reply:
ok, sounds good.